src/evaluator/index.js

```diff
@@ -246,10 +257,12 @@ module.exports = {
    * @memberof radspec/evaluator
    * @param {radspec/parser/AST} ast The AST to evaluate
    * @param {radspec/Bindings} bindings An object of bindings and their values
-   * @param {?string} ethNode The URL to an Ethereum node
+   * @param {?Object} options An options object
+   * @param {?string} options.ethNode The URL to an Ethereum node
+   * @param {?string} options.to The destination address for this expression's transaction
    * @return {string}
    */
-  evaluate (ast, bindings, ethNode) {
-    return new Evaluator(ast, bindings, ethNode).evaluate()
+  evaluate (ast, bindings, options) {
+    return new Evaluator(ast, bindings, options).evaluate()
```

Comment:
Note that we will have to pass `ethNode` in an options object rather than directly when used from aragon.js and the CLI when updating

Reply:
This is a breaking API change so this will be a major version bump.